htdocs/index.php

```diff
@@ -16,7 +16,7 @@
 $tpl->menuitem = MNU_START;
 
 $tpl->caching = true;
-$tpl->cache_lifetime = 300;
+$tpl->cache_lifetime = 60;
```

Comment:
Why change the global cache lifetime?

Reply:
That's too much for an index page. We update a lot of data here. A caching to 60 seconds is sufficient for a highly frequented page. New content will only appear after 5 minutes. The newly installed core_hq_message is also time-controlled and would stay there too long.